### PR TITLE
Make rita.py check the installed routes

### PR DESCRIPTION
**Pull request by drozdziak1**
This commit makes rita.py ensure that optimum routes are installed when checking convergence.

### CODE REVIEW

File: integration-tests/rita.py [hunk 1]
```diff
@@ -323,6 +375,98 @@ def test_reach_all(self, verbose=True, global_fail=True):
...
+        # all_routes = {
+        #     <where_from>: [
+        #                 (<where_to>, <price>, <next_hop>),
+        #                 [...]
+        #             ],
+        #     [...]
+        # }
+
+        all_routes = {
+                self.nodes[1]: [
```
Comment (kingoflolz) [inline integration-tests/rita.py]: I think this shouldn't be self to be consistent with the rest of the dictionary

File: integration-tests/rita.py [hunk 2]
```diff
@@ -414,33 +558,36 @@ def check_log_contains(f, x):
...
-    a = Node(1, 10)
-    b = Node(2, 25)
-    c = Node(3, 60)
-    d = Node(4, 10)
-    e = Node(5, 0)
-    f = Node(6, 50)
-    g = Node(7, 10)
-    # h = Node(8, 0)
-
+    a1 = Node(1, 10)
```
Comment (kingoflolz) [inline integration-tests/rita.py]: any reason they got changed to letter-number? did it get too confusing lol

**Comment by jkilpatr** [comment]
nice, so we went with hardcoding? looks like a clean way to do it at least.

File: integration-tests/rita.py (hunk 1 above)
Comment (drozdziak1) [reply integration-tests/rita.py]: True, thanks!

File: integration-tests/rita.py (hunk 2 above)
Comment (drozdziak1) [reply integration-tests/rita.py]: Yeah, didn't want to wonder which IP a node has every time

**Pull request merged**